### PR TITLE
hotfix(0.18.1): merge into master

### PR DESCRIPTION
**Pull request by seaerchin**
duplicate of #674

### CODE REVIEW

File: src/routes/v2/authenticated/review.ts [hunk 1]
```diff
@@ -70,6 +96,16 @@ export class ReviewsRouter {
     const { userWithSiteSessionData } = res.locals
     const { siteName } = req.params
 
+    const site = await this.sitesService.getBySiteName(siteName)
+    const doesUnmigratedSiteExist = await this.doesUnmigratedSiteExist(
+      userWithSiteSessionData
+    )
+
+    console.log("DOES IT", doesUnmigratedSiteExist)
+
```
Comment (alexanderleegs) [inline src/routes/v2/authenticated/review.ts:105]: Extra log statement here!

File: src/routes/v2/authenticated/review.ts [hunk 2]
```diff
@@ -924,19 +979,24 @@ export class ReviewsRouter {
...
+    // Having session data is proof that this user exists
+    // as otherwise, they would be rejected by our middleware
+    const { userWithSiteSessionData } = res.locals
     const site = await this.sitesService.getBySiteName(siteName)
+    const doesUnmigratedSiteExist = await this.doesUnmigratedSiteExist(
+      userWithSiteSessionData
+    )
+
+    if (!site && doesUnmigratedSiteExist)
+      return res.status(200).json({ message: "Unmigrated site" })
```
Comment (alexanderleegs) [inline src/routes/v2/authenticated/review.ts:992]: I think we might not need the check here, since this is a POST?

**Review by alexanderleegs — COMMENTED** [review]
minor nits

File: src/routes/v2/authenticated/review.ts (hunk 2 above)
Comment (seaerchin) [reply src/routes/v2/authenticated/review.ts]: removed

File: src/routes/v2/authenticated/review.ts (hunk 1 above)
Comment (seaerchin) [reply src/routes/v2/authenticated/review.ts]: removed

**Review by harishv7 — APPROVED** [review]

**Pull request merged**